### DISCUSSION TITLE
Problemas con envío de respuestas en formulario externo

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#506] [4. Bug Reports] [answered]
Hola a todos

Un cliente me reportó que el formulario externo no está funcionando y genera un error desconocido, ahora realicé la prueba y efectivamente me arroja ese error, no estoy logueado con una cuenta de cotalker, ya que la opción del formulario externo es abierta para todo el público. Están en fase de cambio cultural en el proceso y me indicaron que es super relevante resolver esto.

Si estoy en cotalker con una cuenta de la compañia no tengo problemas

Adjunto 2 SS que saqué con el error, genera un error multi pero no sé qué es.

![image](https://user-images.githubusercontent.com/61419471/138485513-26c646a9-adc2-4853-b0ba-076a2cb95913.png)

![image](https://user-images.githubusercontent.com/61419471/138485524-d3c80ddf-529d-4209-ad5a-33384529841a.png)

![image](https://user-images.githubusercontent.com/61419471/138485651-291b895d-fe24-4e46-ac57-90a38fcb11ef.png)

![image](https://user-images.githubusercontent.com/61419471/138487539-e4483574-72bb-47ea-9fc7-d383210f7edc.png)

Link externo: https://web.cotalker.com/public-survey/5fb272302f5af2000898d992/60e74bc949ed5400090c2ad8

@jdurandpaz y @guillevalin favor su ayuda con esto.

gracias =>

**Comment by acedward** [comment, marked as answer]
Gracias @NicolasPLG efectivamente es un bug. Ya lo hemos detectado, vamos a corregirlo y mandar un hotfix durante el día.

----
Error 400 {"data":null,"error":{"code":"BAD_REQUEST","message":"Bad request"}}

---- 
I [2021-10-22T18:00:54.691Z] POST - /api/v2/messages/multi output: {   "message": "Cannot read property 'checkMessageCmdMeta' of undefined" } body: {   "meta": {     "action": "create-task",     "payload": {       "group": "...."     }   },   "cmd": [     {       "method": "POST",       "_id": "....",       "message": {         "_id": ".....",         "content": "",         "contentArray": [           {             "_id": ".....",             "command": {               "commands": [],               "resetIdentifiers": [],               "isCommanded": null,               "values": []             },             "display": [               "## Ingreso de solicitud de contrato"             ],             "code": [],             "isActive": true,             "isSystemModel": false,             "contentType": "application/vnd.cotalker.survey+text",             "help": null,             "identifier": "ingreso_solicit

**Reply by jdurandpaz** [reply]
gracias @acedward es crítico para que el cliente comience la utilización del sistema

**Reply by acedward** [reply]
Debería estar funcional

**Reply by acedward** [reply]
![image](https://user-images.githubusercontent.com/3951895/138507819-19401aad-2aa2-4320-84ab-3de791bbebe3.png)

**Reply by NicolasPLG** [reply]
gracias @acedward  lo probaré antes de informar al cliente.

**Reply by NicolasPLG** [reply]
funciona muchas gracias, ya informe al cliente.

Cierro la discusión
